examples/blazor/Pages/Index.razor

```diff
@@ -0,0 +1,47 @@
+﻿@page "/"
+@using System.Timers;
+
+<div class="container">
+  <div class="col-md-8 offset-md-2">
+    <h4 class="mb-3">Contact</h4>
+    <form>
+      <div class="row">
+          <fluent-text-field @bind-value="FirstName" @bind-value:event="onchange">First Name</fluent-text-field>
+      </div>
+      <div class="row">
+          <fluent-text-field @bind-value="LastName" @bind-value:event="onchange">Last Name</fluent-text-field>
+      </div>
+      <div class="row">
+          <fluent-text-field id="username">Username</fluent-text-field>
+      </div>
+      <div class="row">
+        <fluent-text-field type="email" id="email">Email</fluent-text-field>
+      </div>
+      <div class="row">
+        <fluent-text-area id="message" placeholder="Please enter your message here.">Message</fluent-text-area>
+      </div>
+      <div class="row">
+        <fluent-checkbox id="sign-up">
+          Sign up for our newsletter
+        </fluent-checkbox>
+      </div>
+      <div class="row">
+        <fluent-button class="call-to-action" appearance="accent">Send</fluent-button>
+      </div>
+    </form>
+  </div>
+</div>
+
+@code {
+  private string FirstName { get; set; } = "Blazor";
+  private string LastName { get; set; } = "FAST";
+
+  protected override void OnInitialized()
+  {
+    new Timer {
+      Interval = 500,
+      AutoReset = true,
+      Enabled = true
+    }.Elapsed += delegate { Console.WriteLine($"{this.FirstName} {this.LastName}"); };
```

Comment:
Not sure what's the idea with this. Is the goal just to show the data was edited? If so it might be simpler to remove the timer and just have some HTML that displays the values on `@FirstName` and `@LastName`.

BTW if you are choosing to keep this timer, you definitely need to dispose it when the component itself is disposed, otherwise you'll leak memory and spam the console forever.